### DISCUSSION TITLE
Two-lined status bar from both sides (above and below)

### DISCUSSION THREAD

**Discussion by voodookiidoo** [erikw/tmux-powerline#451] [Ideas]
The fact that i can make status bar split in two is awesome. I wonder if it would be possible to make them appear on different sides?
For example - segments could be shown below the main window, and windows pane could be shown above it

**Comment by erikw** [comment]
For now, the supported modes are these:
https://github.com/xx4h/tmux-powerline/blob/56308f3b2a5488ce2549b12f7f818f5f786def4b/lib/config_file.sh#L120
